### PR TITLE
Remove non-semantic changes prior to using a sequenced threadpool in …

### PR DESCRIPTION
**Pull request by baldersheim**
…mbus.

@vekterli or @geirst PR

### CODE REVIEW

**Review by geirst — APPROVED** [review]
👍

File: storageapi/src/vespa/storageapi/mbusprot/storagecommand.h [hunk 1]
```diff
@@ -21,6 +21,8 @@ class StorageCommand : public mbus::Message, public StorageMessage {
     api::StorageMessage::SP getInternalMessage() override { return _cmd; }
     api::StorageMessage::CSP getInternalMessage() const override { return _cmd; }
 
+    bool hasBucketSequence() const override { return false; }
```
Comment (vekterli) [inline storageapi/src/vespa/storageapi/mbusprot/storagecommand.h:24]: If we'll be using this to signal sequencing, should it also be explicitly added to `StorageReply`? Both commands and replies need sequencing for storage API stuff

**Review by vekterli — COMMENTED** [review]
:+1:

File: storageapi/src/vespa/storageapi/mbusprot/storagecommand.h (hunk 1 above)
Comment (baldersheim) [reply storageapi/src/vespa/storageapi/mbusprot/storagecommand.h:24]: Yes, probably. I will try and see what it is used for currently first.

**Pull request merged**